.github/workflows/lint.yml

```diff
@@ -8,20 +8,16 @@ on:
 permissions: read-all
 
 jobs:
-  flake8-lint:
+  whitespace-lint:
     runs-on: ubuntu-latest
-    name: Lint
+    name: Lint whitespace
     steps:
       - name: Check out source repository
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
       - name: Set up Python environment
         uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
         with:
           python-version: "3.11"
```

Comment:
we should be able to remove actions-setup/python after removing flake8?

Reply:
Good idea. Removed